run/rclone_archive/archive-clips.sh

```diff
@@ -4,15 +4,51 @@ log "Moving clips to rclone archive..."
 
 source /root/.teslaCamRcloneConfig
 
-FILE_COUNT=$(find "$CAM_MOUNT"/TeslaCam/SavedClips/* -type f | wc -l)
-
-rclone --config /root/.config/rclone/rclone.conf move "$CAM_MOUNT"/TeslaCam/SavedClips "$drive:$path" --create-empty-src-dirs --delete-empty-src-dirs >> "$LOG_FILE" 2>&1 || echo ""
-
-FILES_REMAINING=$(find "$CAM_MOUNT"/TeslaCam/SavedClips/* -type f | wc -l)
-NUM_FILES_MOVED=$((FILE_COUNT-FILES_REMAINING))
-
-log "Moved $NUM_FILES_MOVED file(s)."
-/root/bin/send-push-message "TeslaUSB:" "Moved $NUM_FILES_MOVED dashcam file(s)."
-
+NUM_FILES=0
+NUM_FILES_MOVED=0
+NUM_FILES_IN_DIR=0
+
+function connectionmonitor {
+  while true
+  do
+    if timeout 5 /root/bin/archive-is-reachable.sh $ARCHIVE_HOST_NAME
+    then
+      sleep 2
+    elif timeout 5 /root/bin/archive-is-reachable.sh $ARCHIVE_HOST_NAME # try one more time
+    then
+      sleep 2
+    else
+      log "connection dead, killing archive-clips"
+      # The network connection may have been lost, so kill it hard.
+      # (should be no worse than losing power in the middle of an operation)
+      kill -9 $1
+      return
+    fi
+  done
+}
+
+function moveclips() {
+  #rclone doesn't remove folders after moving files. This deletes empty directories
+  find "$CAM_MOUNT"/TeslaCam/SavedClips/ -type d -empty -delete
+
+  for file_name in "$CAM_MOUNT"/TeslaCam/saved* "$CAM_MOUNT"/TeslaCam/SavedClips/*; do
```

Comment:
This is the old code, which was recently replaced. I'd like to keep the new code and not go back to the old code.